internal/test/e2e/e2e_test.go

```diff
@@ -48,16 +49,10 @@ var (
 	fUseKind           = flag.Bool("kind", false, "Creates a Kind cluster to run the tests against.")
 	fOutputDirectory   = flag.String("output-directory", "/tmp/etcd-e2e", "The absolute path to a directory where E2E results logs will be saved.")
 	fKindClusterName   = flag.String("kind-cluster-name", "etcd-e2e", "The name of the Kind cluster to use or create")
+	fControllerImage   = flag.String("controller-image", "", "The name of the controller image")
```

Comment:
This allows me to supply the VERSION tagged image name from the Makefile so that that exact image can be exported and loaded into kind (below). But as noted above, I think it'd be better if we removed all the creation of kind cluster from this file and did it in the Makefile instead....a future enhancement though.